src/epcpm/importexportdialog.py

```diff
@@ -123,28 +123,28 @@ def paths_from_directory(directory):
-        tables_c                  = interface_path / "canInterfaceGenTables.c",
-        bitfields_c               = interface_path / "interfaceBitfieldsGen.c",
-        staticmodbus_c            = interface_path / "staticmodbusInterfaceGen.c",
-        sunspec1_interface_gen_c  = sunspec_path / "sunspec1InterfaceGen.c",
-        sunspec2_interface_gen_c  = sunspec_path / "sunspec2InterfaceGen.c",
-        sunspec1_tables_c         = sunspec_path / "sunspec1InterfaceGenTables.c",
-        sunspec2_tables_c         = sunspec_path / "sunspec2InterfaceGenTables.c",
-        sunspec1_spreadsheet      = embedded_path / "MODBUS_SunSpec1-EPC.xlsx",
-        sunspec2_spreadsheet      = embedded_path / "MODBUS_SunSpec2-EPC.xlsx",
-        sunspec1_spreadsheet_user = embedded_path / "EPCSunspec1.xlsx",
-        sunspec2_spreadsheet_user = embedded_path / "EPCSunspec2.xlsx",
-        staticmodbus_spreadsheet  = embedded_path / "MODBUS-EPC.xlsx",
-        smdx                      = sorted(sunspec_path.glob("smdx_*.xml")),
-        sunspec_c                 = sunspec_path,
-        sil_c                     = path / "sil" / "libEpcControlInterfaceGen.c",
-        interface_c               = interface_path / "interfaceGen.c",
-        rejected_callback_c       = interface_path / "rejectedCallbackHandler.c",
-        spreadsheet_can           = interface_path / "EPC-CAN.xlsx",
-        anomalies_h               = interface_path / "anomalies_generated.h",
-        anomalies_spreadsheet     = interface_path / "anomalies.xlsx",
+        can=devices_path / "EPC_ID247.sym",
+        hierarchy=devices_path / "EPC_ID247.hierarchy.json",
+        tables_c=interface_path / "canInterfaceGenTables.c",
+        bitfields_c=interface_path / "interfaceBitfieldsGen.c",
+        staticmodbus_c=interface_path / "staticmodbusInterfaceGen.c",
+        sunspec1_interface_gen_c=sunspec_path / "sunspec1InterfaceGen.c",
+        sunspec2_interface_gen_c=sunspec_path / "sunspec2InterfaceGen.c",
+        sunspec1_tables_c=sunspec_path / "sunspec1InterfaceGenTables.c",
+        sunspec2_tables_c=sunspec_path / "sunspec2InterfaceGenTables.c",
+        sunspec1_spreadsheet=embedded_path / "MODBUS_SunSpec1-EPC.xlsx",
+        sunspec2_spreadsheet=embedded_path / "MODBUS_SunSpec2-EPC.xlsx",
+        sunspec1_spreadsheet_user=embedded_path / "EPCSunspec1.xlsx",
+        sunspec2_spreadsheet_user=embedded_path / "EPCSunspec2.xlsx",
+        staticmodbus_spreadsheet=embedded_path / "MODBUS-EPC.xlsx",
+        smdx=sorted(sunspec_path.glob("smdx_*.xml")),
+        sunspec_c=sunspec_path,
+        sil_c=path / "sil" / "libEpcControlInterfaceGen.c",
+        interface_c=interface_path / "interfaceGen.c",
+        rejected_callback_c=interface_path / "rejectedCallbackHandler.c",
+        spreadsheet_can=interface_path / "EPC-CAN.xlsx",
+        anomalies_h=interface_path / "anomalies_generated.h",
+        anomalies_spreadsheet=interface_path / "anomalies.xlsx",
```

Comment:
Revert this.

Reply:
Leave it. The previous code is not PEP-8 compliant. CI should fail, so I'm surprised these lines were formatted this way in the first place.

Dave, we use `black` to format all python code so that developers don't have to concern themselves with formatting style anymore.

Reply:
* "concern themselves with legibility", FTFY.

Reply:
anyway... I only commented here when I somehow mistook this for part of the formatted segments used in the generated code. You guys can do what you want with your Python formatting.